src/go/rpk/pkg/cli/cmd/debug/bundle/bundle_k8s_linux.go

```diff
@@ -307,3 +350,63 @@ func requestAndSave[T1 any](ctx context.Context, ps *stepParams, filename string
 	}
 	return nil
 }
+
+func parseJournalTime(t string) (time.Time, error) {
+	/*
+		From `man journalctl`:
+			Date specifications should be of the format "2012-10-30 18:17:16". If the time part is omitted, "00:00:00" is assumed.
+			If only the seconds component is omitted, ":00" is assumed. If the date component is omitted, the current day is assumed.
+			Alternatively the strings "yesterday", "today", "tomorrow" are understood, which refer to 00:00:00 of the day before
+			the current day, the current day, or the day after the current day, respectively. "now" refers to the current time.
+			Finally, relative times may be specified, prefixed with "-" or "+", referring to times before or after the current
+			time, respectively.
```

Comment:
Nit on length: I think this is _very_ wide, can this drop two levels of tabbing, and then wrap the text to the standard 80 cols?